sdk/storage/Azure.Storage.Blobs.Cryptography/src/Azure.Storage.Blobs.Cryptography.csproj

```diff
@@ -17,6 +17,7 @@
       Microsoft Azure Storage REST API Reference - https://docs.microsoft.com/en-us/rest/api/storageservices/
       REST API Reference for Blob Service - https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api
     </Description>
+    <RootNamespace>Azure.Storage.Blobs.Specialized.Cryptography</RootNamespace>
```

Comment:
Is there a reason we're not putting everything in `.Specialized`?

Reply:
Response in previous comment on this topic.